config/manager/config.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: numaplane-controller-config
+  namespace: numaplane-system
+data:
+  config.yaml: |
+    clusterName: "example-cluster"
+    timeIntervalSec: 60
+    repoCredentials:
```

Comment:
Let's take the `repoCredentials` out of here. This represents our default manifest which we will deploy. 

To be honest, I think we ultimately will need to even remove `clusterName` from here and expect that that will be added by the kustomization which is done outside of this repo, but for now let's make it so that [this](https://github.com/numaproj-labs/numaplane/blob/main/Makefile#L149) line works. See how it's configured as an environment variable in the `Deployment` object [here](https://github.com/numaproj-labs/numaplane/blob/main/config/manager/manager.yaml#L75)? Let's remove it from there and instead set `clusterName: CLUSTER_NAME_VALUE` here

Reply:
Removing this will cause the loadConfig fail as it needs all the keys .